torch/_export/utils.py

```diff
@@ -9,26 +9,33 @@
 from torch.utils._pytree import (
     _register_pytree_node,
     Context,
+    DumpableContext,
     FlattenFunc,
-    MaybeFromStrFunc,
-    ToStrFunc,
+    FromDumpableContextFn,
+    ToDumpableContextFn,
     UnflattenFunc,
 )
 
 
+SERIALIZED_DATACLASS_TO_PYTHON_DATACLASS: Dict[str, Type[Any]] = {}
+
+
```

Comment:
mmm I feel uncomfortable that exporting a model has a side-effect of making the model's input and output dataclasses into pytrees. Ideally this would be localized to export APIs only. But this is a pre-existing condition so don't worry about it

Reply:
NVM, yanan corrected me.